pomegranate/gmm.pyx

```diff
@@ -167,17 +170,22 @@ cdef class GeneralMixtureModel(BayesModel):
 			else:
 				improvement = log_probability_sum - last_log_probability_sum
 
+				time_spent = time.time() - epoch_start_time
+				vals = dict(iteration=iteration, improvement=improvement, time=time_spent)
+				fmt = "[{iteration}] Improvement: {improvement} Time (s): {time:.2f}"
```

Comment:
@jmschrei I realize you may find this to be a bit too much, but I believe that it is better to create a variable this way than trying to fit it into the `print` line below, which would then overflow.